extension/src/config/constants.js

```diff
@@ -23,6 +23,16 @@ module.exports = {
   ],
 
   PAYMENT_METHOD_TYPE_AFFIRM_METHODS: ['affirm'],
+  PAYMENT_METHODS_REQUIRES_LINE_ITEMS: [
```

Comment:
Where did you get this list? In the link above I see different payment method names. Is it the same ones? https://docs.adyen.com/api-explorer/#/CheckoutService/latest/payments__reqParam_lineItems

Reply:
I checked one by one for all payment methods from the list: https://docs.adyen.com/online-payments/supported-payment-methods and noted the ones that requires lineItems.